src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11685,8 +11754,27 @@ namespace UnitarySystems {
                 LatentLoad = false;
             }
 
-            // disable the coil if the condition is NOT favorable
-            if (this->m_TemperatureOffsetControlActive) {
+            // activate heat recovery loop coil if scheduled on and there is air flow
+            if (this->m_WaterHRPlantLoopModel) {
+                if (this->temperatureOffsetControlStatus == 1) {
+                    PartLoadFrac = 1.0;
+                    mdot = this->MaxCoolCoilFluidFlow;
+                }
+                PlantUtilities::SetComponentFlowRate(state,
+                                                     mdot,
+                                                     this->CoolCoilFluidInletNode,
+                                                     this->CoolCoilFluidOutletNodeNum,
+                                                     this->CoolCoilLoopNum,
+                                                     this->CoolCoilLoopSide,
+                                                     this->CoolCoilBranchNum,
+                                                     this->CoolCoilCompNum);
+
+                WaterCoils::SimulateWaterCoilComponents(
+                    state, CompName, FirstHVACIteration, this->m_CoolingCoilIndex, _, this->m_FanOpMode, PartLoadFrac);
+                SensibleLoad = false; // fall through remaining checks
+                LatentLoad = false;
+            } else if (this->m_TemperatureOffsetControlActive) {
```

Comment:
The above is basically, if the coil is configured as heat recovery and is active set the PartLoadFrac to 1, set mdot to full flow, and call the coil model, then bypass other calcs in this function.

Reply:
Will the coil model limit mdot to the available flow on the branch?

Reply:
Yes, based on the SetComponentFlowRate call just above the SimulateWaterCoilComponents call.

Reply:
yes, the SetComponentFlowRate is there to impose limits and what is actually available.  It can change the value of Mdot to what plant really can do.